src/internal/connector/exchange/api/shared.go

```diff
@@ -83,7 +86,13 @@ func getItemsAddedAndRemovedFromContainer(
 			return nil, nil, "", err
 		}
 
-		logger.Ctx(ctx).Infow("Got page", "items", len(items))
+		itemCount += len(items)
+		page++
+
+		// Log every ~1000 items (the page size we use is 200)
+		if page%5 == 0 {
```

Comment:
Shouldn't the mod be calculated for the configured value (200) for when we change it?